pages/linux/genid.md

```diff
@@ -9,7 +9,7 @@
 
 - Generate a UUIDv5 using a namespace UUID and a specific name:
 
-`genid uuidv5 {{{ce598faa-8dd0-49ee-8525-9e24fff71dca}}} {{name}}`
+`genid uuidv5 {{ce598faa-8dd0-49ee-8525-9e24fff71dca}} {{name}}`
```

Comment:
Can someone on Windows with mingw check this please? I suspect this might be a bug.

Reply:
Kinda curious, isn't this a Linux-specific page, is testing on Windows required (if so I can do it now)?